pkgs/top-level/all-packages.nix

```diff
@@ -15910,6 +15910,8 @@ with pkgs;
 
   thinkingRock = callPackage ../applications/misc/thinking-rock { };
 
+  nylas-mail = callPackage ../applications/networking/mailreaders/nylas-mail { };
```

Comment:
Whoops. Thanks.

One other thing, this is my first time creating a [module](https://github.com/johnramsden/nixpkgs/blob/c38207983a04c3a6a17ce77e854ce2ffde207283/nixos/modules/programs/nylas-mail.nix) for a package - someone might want to look it over to make sure everything looks alright.